torch/fx/symbolic_trace.py

```diff
@@ -119,34 +113,31 @@ def is_leaf_module(self, m: torch.nn.Module, module_qualified_name : str) -> boo
         """
         return m.__module__.startswith('torch.nn') and not isinstance(m, torch.nn.Sequential)
 
+    def get_module_qualified_name(self, mod):
```

Comment:
Can you describe the case where this logic doesn't work and needs to be overridden? I want to understand more about why we are not finding the module by searching the module hierarchy for it.

Reply:
nit: `path_of_module` might be a better name because torch.nn.Linear.__qualname__ exists and can be mistaken for the qualified name of a module..

Reply:
> Can you describe the case where this logic doesn't work and needs to be overridden? I want to understand more about why we are not finding the module by searching the module hierarchy for it.

An example is internal dper3 code, where modules are wrapped by PyTorchModuleWrapper. Please see how this function is reimplemented in D24239120.